xcodeproj.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'claide',         '>= 1.0.2', '< 2.0'
   s.add_runtime_dependency 'colored2',       '~> 3.1'
   s.add_runtime_dependency 'nanaimo',        '~> 0.3.0'
+  s.add_runtime_dependency 'rexml',          '~> 3.2.4'
```

Comment:
where is this used? I cant tell from the diff

Reply:
It's used for reading/writing schemes. Not in the diff because it was a "default" (always installed) gem (or part of the stdlib) before 3.0

Reply:
nice!